### ISSUE TITLE
Alternative Download for APK

### ISSUE THREAD

**Issue by IzzySoft** [TheAlphaApp/flutter_riverpod_todo_app#11]
Will you consider an alternative Download for users outside Google's walled garden – e.g. attaching APKs to [releases](https://github.com/TheAlphaApp/flutter_riverpod_todo_app/releases) here? I'm thinking of users without a Playstore account and even without Play Services on their devices, either by their own choice (privacy reasons etc), or because the manufacturer ships its devices that way (think e.g. Huawei). Thanks in advance!

**Comment by TheAlphaApp** [comment]
Sure. I'll do this whenever I'll be free. 
It would be great if you can create a pull request.

**Comment by IzzySoft** [comment]
I'm not a dev, sorry, so I've no idea what I should push. But what I can offer, once the APK is there, is to make it available [via my repo](https://apt.izzysoft.de/fdroid) to make it easy to access (including automated updates) using an [F-Droid client](https://android.izzysoft.de/applists/category/named/apps_markets#group_1181) of the user's choice. If you want to go ahead for [F-Droid.org](https://f-droid.org/) itself, I can also help with that.

**Comment by TheAlphaApp** [comment]
I've released a new version [here](https://github.com/TheAlphaApp/flutter_riverpod_todo_app/releases/tag/v1.2.4). I also included apks in the release assets. Can I close this issue?

**Comment by TheAlphaApp** [comment]
I don't have any experience with the F-droid. I don't know how F-droid client works. In case you don't know, I want to tell you that `app-release.apk` is a “fat” APK which includes all target architectures. I think you should consider `app-release.apk` as default instead of `armeabi-v7a`. Finally, it's your choice.

**Comment by IzzySoft** [comment]
Yes, I'm aware of the "fat releases". The reason I usually pick ARMv7 instead is size: there's a limit of 30M reserved per-app with my repo. If the fat APK is smaller than 10M I gladly take that (so 3 versions fit in); up to 15M I consider it (as 2 versions fit in). The reason I prefer multiple versions to fit is "user experience": it's simply frustrating when installations fail – which easily happens once a new release was pushed but the user had not yet updated the index on his client. Having an overlap, the previous version would be picked then – and updated the next day. In your case, the "fat build" is 18M+, so only 1 version would fit into the limit and this "incident" becomes likely.

You might ask why there's that limit. The entire repo etc. runs on my personal space, so I have to limit it. It currently serves 830+ apps, which already sums up a bit (close to 10G at this time). And I have to reserve that space locally plus online :wink:

**Comment by TheAlphaApp** [comment]
Okay, You're doing great. Thank you. I'm closing this.
